src/Controller/ForgotMyPasswordController.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\User;
+use App\Form\ForgotMyPasswordType;
+use App\Form\ResetPasswordType;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\Email;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Component\Routing\RouterInterface;
+use Symfony\Component\Uid\Uuid;
+
+/**
+ * @Route(path="/users")
+ */
+class ForgotMyPasswordController extends AbstractController
+{
+    private EntityManagerInterface $entityManager;
+
+    private RouterInterface $router;
+
+    private MailerInterface $mailer;
+
+    public function __construct(
+        EntityManagerInterface $entityManager,
+        RouterInterface $router,
+        MailerInterface $mailer
+    ) {
+        $this->entityManager = $entityManager;
+        $this->router = $router;
+        $this->mailer = $mailer;
+    }
+
+    /**
+     * @Route(path="/forgot-password")
+     */
+    public function new(Request $request): Response
+    {
+        $task = new User();
```

Comment:
What do you think about the name of this variable?

Reply:
Renamed it.